content/automate/ManageIQ/Cloud/Orchestration/Retirement/StateMachines/Methods.class/__methods__/start_retirement.rb

```diff
@@ -17,7 +17,7 @@
   exit MIQ_ABORT
 end
 
-if stack.retiring?
+unless stack.retirement_initialized
   $evm.log('error', "Stack is in the process of being retired. Aborting current State Machine.")
```

Comment:
@d-m-u Let's change the error message to something like: "Stack has not been initialized for retirement. Aborting current State Machine."